pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
     "Microsoft.CSharp": "4.3.0",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.3-servicing-25319-01",
+    "Microsoft.NETCore.Runtime.CoreCLR": "2.0.0-preview2-25319-01",
```

Comment:
This update came from the broken CoreCLR build, and the old Core-Setup auto-update allows multiple package entries with the same ID, unlike CoreFX/CoreCLR auto-upgrade which both broke. The Core-Setup upgrade should be fine when the next CoreCLR build comes out.